src/app/rpmostree-builtin-compose.c

```diff
@@ -32,6 +32,15 @@ static RpmOstreeCommand compose_subcommands[] = {
   { "tree", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
     "Install packages and commit the result to an OSTree repository",
     rpmostree_compose_builtin_tree },
+  { "install", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
+    "Install packages into a target path",
+    rpmostree_compose_builtin_install_root },
+  { "postprocess", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
```

Comment:
Can you expand on the value of making this its own command? In what case would users want to run their scripts/ansible after postprocess rather than before?

Or is this basically to enable e.g. bundling up the rootfs into whatever format we want for transport and deferring the commit step?

Reply:
Right, we want to support e.g. repacking the repo into OCI.  One issue is that `rpm-ostree compose commit` is then a *really* thin wrapper around `ostree commit`...in my prototype `skopeo2ostree` script I find it easier just to commit directly.